helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -387,12 +396,21 @@ spec:
           name: {{ template "splunk-otel-collector.fullname" . }}-fluentd-json
       {{- end}}
       {{- if eq .Values.logsEngine "otel" }}
+      {{- if .Values.isWindows }}
+      - name: varlog
+        hostPath:
+          path: C:/var/log
+      - name: varlibdockercontainers
+        hostPath:
+          path: C:/ProgramData
```

Comment:
Can you please clarify why the whole `C:/ProgramData` has to be mounted to container? What is it being used for?

Reply:
@dmitryax 
- When we have `containerRuntime: docker` at that time docker Daemon’s container storage location will be `C:/ProgramData/docker/containers`
- When we have `containerRuntime: containerd` at that time container storage location will be different then docker runtime and it will be something like `C:/ProgramData/containerd/`

Note: I have taken the reference of DataDog helm chart from [hear](https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/_container-agent.yaml#L180-L182)

Reply:
Interesting. Ok please add a comment line saying that `C:/ProgramData/docker/containers` and `C:/ProgramData/containerd/root` are being used. Also please verify that it doesn't break metrics collection. Otherwise looks good

Reply:
Yes, I have checked metrics collection also it is not affecting.
Observability Chart:
<img width="1792" alt="Screenshot 2022-01-20 at 10 26 47 AM" src="https://user-images.githubusercontent.com/88535006/150276128-73df1ba5-f0b2-4d38-b727-948a34ba2505.png">

Reply:
Which metric is on the screenshot? It looks like a clusterReceiver metric. We need kubelet metrics. Please check k8s navigator https://app.signalfx.com/#/infra/k8s/map

Reply:
yes that was hostmetrics here you can find the kubeletstats metrics also being collected.
k8s navigator:
<img width="1792" alt="Screenshot 2022-01-20 at 10 47 40 AM" src="https://user-images.githubusercontent.com/88535006/150278211-0aecb1b7-f217-4926-96d5-5a85052dc607.png">

Reply:
Looks good, thanks for validating it. Please add a comment to C:/ProgramData volume that I mentioned above and this should be good to go

Reply:
Added comment